security/backends/elasticsearch/tests.py

```diff
@@ -0,0 +1,25 @@
+from uuid import uuid4
+
+from django.test.utils import override_settings, TestContextDecorator
+
+from .models import CommandLog, CeleryTaskRunLog, CeleryTaskInvocationLog, InputRequestLog, OutputRequestLog
+
+
+class store_elasticsearch_log(override_settings):
```

Comment:
helper for project testing. We need separate index for parallel test processes. Therefore this context processor create new index for every log and remove it int the end of the test/block